modules/imaging_browser/help/viewsession.md

```diff
@@ -0,0 +1,60 @@
+# Imaging Browser: View Session
+
+The View Session page displays a candidate's scans collected from
+a single timepoint.  General information on a candidate's session
+appears in a table at top including Patient Name, PSCID, DCCID,
+Visit Label, etc.
+
+Files for the selected candidate's dataset are displayed below the
+general information table. Horizontal (axial), sagittal and coronal
+thumbnails are displayed -- click on any image to launch the
+BrainBrowser visualization tool.
+
+To the right of the image thumbnails appears a list of scan parameters
+including QC Status, "Selected" (indicating which scan will be
+available in the data query tool when there are multiple scans of
+a single modality), and a Caveat flag (True/False). Within the same
+panel, users may click "Header Info" located in the top right corner
+to expand the panel, revealing further scan parameters from the
+DICOM header such as Voxel size, Acquisition Date, Protocol, Echo
+Time etc. To toggle this panel and hide this information, click
+"Header Info" again.
+
+Buttons at the bottom of each scan panel allow users to view/edit
+QC Comments, download the scan as a Minc file, or viewing the scans
+from a Longitudinal View which will launch the BrainBrowser. Click
+the QC Comments button to open up a pop-up window in which users
+with appropriate permissions may select QC issues and enter comments
+regarding intensity, movement artifacts, coverage and overall
+feedback on the selected scan.
+
+In the left sidebar
```

Comment:
@driusan 

just a small formatting issue here; should this "In the left sidebar" be a main heading/title?

Reply:
I had just copy/pasted it from the other PR, but I agree.. I most it to be part of the title.